File: lab/JAVA/Combinador.java
import java.util.*;


class Combinador {
	public static void main(String[] args) {
	  String str1 = new String();
	  String str2 = new String();
	  String str3 = new String();
	  Scanner sc = new Scanner(System.in);
	  int len = 0;
	  int k = 0; //indice str1
	  int j = 0; //indice str2
	  int l = 0; //indice str3

	  while(sc.hasNext()){
	    str1 = sc.next();
	    str2 = sc.next();
	    len = str1.length() + str2.length();
	    char[] arrChar = new char[len];

	    for(int i = 0; i <= len; i = i + 1){
	      if(i < str1.length()){
		arrChar[l] = str1.charAt(k);
		l = l + 1;
		k = k + 1;
	      }

	      if(i < str2.length()){
	       arrChar[l] = str2.charAt(j);
	       l = l + 1;
	       j = j + 1;
	      }

	    }

	    str3 = new String(arrChar);

	    //System.out.println(str1);
	    //System.out.println(str2);
	    System.out.println(str3);
	  }

	  sc.close();
	}
}

File: lab/C/combinador.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main(void){
  char str1[255];
  char str2[255];
  int len = 0;

  while(scanf("%s %s", str1, str2) != EOF){
    int l = 0; int j = 0; int k = 0;
    int len = strlen(str1) + strlen(str2);
    char str3[len + 1];
    for(int i = 0; i < len; i = i + 1){
      if(i < strlen(str1)){
	 str3[l++] = str1[k++];
      }

      if(i < strlen(str2)){
	str3[l++] = str2[j++];	
      }
    }

    for(int i = 0; i < len; i = i + 1){
      printf("%c", str3[i]);
    }

    printf("\n");
  }

  return 0;
}
